m4/cf3_platforms.m4

```diff
@@ -39,3 +39,4 @@ AM_CONDITIONAL([HPUX], [test -n "`(echo ${target_os} | egrep 'hpux|hp-ux')`"])
 AM_CONDITIONAL([FREEBSD], [test -n "`(echo ${target_os} | grep freebsd)`"])
 AM_CONDITIONAL([NETBSD], [test -n "`(echo ${target_os} | grep netbsd)`"])
 AM_CONDITIONAL([XNU], [test -n "`(echo ${target_os} | grep darwin)`"])
+AM_CONDITIONAL([SHUNIT], [test -n "`which shunit2`"])
```

Comment:
The tests actually work on any POSIX platform that has shunit available (I did most of the development for this script on OS X). If they don't I would consider that a bug, as I tried very hard to avoid bashisms and only use portable things :)